src/components/PageComponents.js

```diff
@@ -14,14 +14,13 @@ class PageComponents extends Component {
 
   constructor(props) {
     super(props)
-    this.state = { activeProvider: 'github' }
+    this.state = { }
```

Comment:
you can just delete setting the state unless it is used for something. No technical harm in having it but points people to think that the component actually has/develops meaningful state.